App.js

```diff
@@ -1,13 +1,32 @@
-import { StatusBar } from 'expo-status-bar';
-import { StyleSheet, Text, View } from 'react-native';
+import { useState } from 'react'
+import { StatusBar } from 'expo-status-bar'
+import { StyleSheet, Text, View, TextInput, Button } from 'react-native'
 
 export default function App() {
+  const [studyKey, setStudyKey] = useState('')
+  const [study, setStudy] = useState()
+
+  async function handleSubmit() {
+    await fetch(`http://localhost:3000/api/studies/key/${studyKey.trim()}`)
```

Comment:
We should store URL in an env var and grab from there so we don't have to change it everywhere in the codebase when we deploy at the end of project.

Reply:
Agreed. I tried to set this up but it's not as straightforward in React Native so I was going to create a separate story. Looks like there are some additional considerations for env variables to work cross-platform. There are libraries like [this one](https://github.com/luggit/react-native-config) though which will hopefully simplify things.

Reply:
Done, ended up going the singleton route since it's simpler and using that React Native Config library doesn't obfuscate the env vars anyway